client/state/sharing/services/test/selectors.js

```diff
@@ -97,6 +98,20 @@ describe( 'selectors', () => {
 		} );
 	} );
 
+	describe( 'getKeyringServiceByName()', () => {
+		it( 'should return empty object if there is no service', () => {
```

Comment:
Should this be "should return ~empty object~`false` if there is no service"? :trollface: